executor/simple.go

```diff
@@ -1100,25 +1122,23 @@ func renameUserHostInSystemTable(sqlExecutor sqlexec.SQLExecutor, tableName, use
 func (e *SimpleExec) executeDropUser(s *ast.DropUserStmt) error {
 	// Check privileges.
 	// Check `CREATE USER` privilege.
-	if !config.GetGlobalConfig().Security.SkipGrantTable {
```

Comment:
This is safe to remove because the `RequestVerification` (and similar) API calls check if SkipGrant is set already. It might be a legacy case from before this was done.